doc/examples/animation_widget/lib/main.dart

```diff
@@ -71,7 +72,7 @@ class _MyHomePageState extends State<MyHomePage> {
             Flame.util.animationAsWidget(
                 _position,
                 animation.Animation.sequenced('minotaur.png', 19,
-                    textureWidth: 96.0)),
+                    textureWidth: 96.0, textureHeight: 96)),
```

Comment:
let's do one param per line or all in the same line (if it fits)

Reply:
i'll add commas